src/lib/Component.js

```diff
@@ -81,11 +82,13 @@ class Component extends HTMLElement {
     this.#isConnected = true
     // Load the DOM
     this.render()
+    this.#customEvents.forEach(([customEvent, listener]) => this.addEventListener(customEvent, listener))
```

Comment:
I am reading more and more that `.forEach` is not recommended due to performance issues (i.e. see [Biome documentation](https://biomejs.dev/linter/rules/no-for-each/)). Should I turn it into `for ... of`?